clients/lsp-angular.el

```diff
@@ -36,13 +36,15 @@
   :link '(url-link "https://github.com/angular/vscode-ng-language-service"))
 
 (defcustom lsp-clients-angular-language-server-command
-  '("node"
```

Comment:
We should not run programs as part of the initialization of a properly since it will run for all users using lsp-mode. The option is to move the calculation in a function so it will run only when the user is actually in an angular project but IMHO better to create an automatic installation for that server.